src/vega.schema.ts

```diff
@@ -501,145 +525,118 @@ export type VgBinding = VgCheckboxBinding | VgRadioBinding |
  * Base object for Vega's Axis and Axis Config.
  * All of these properties are both properties of Vega's Axis and Axis Config.
  */
-export interface VgAxisBase {
+export interface VgAxisConfig {
   /**
-   * A boolean flag indicating if the domain (the axis baseline) should be included as part of the axis.
+   * An interpolation fraction indicating where, for `band` scales, axis ticks should be positioned. A value of `0` places ticks at the left edge of their bands. A value of `0.5` places ticks in the middle of their bands.
    *
-   * __Default value:__ `true`
+   *  __Default value:__ `0.5`
    */
-  domain?: boolean;
+  bandPosition?: number;
 
   /**
-   * A boolean flag indicating if grid lines should be included as part of the axis
+   * A boolean flag indicating if the domain (the axis baseline) should be included as part of the axis.
    *
-   * __Default value:__ `true` for [continuous scales](https://vega.github.io/vega-lite/docs/scale.html#continuous) that are not binned; otherwise, `false`.
+   * __Default value:__ `true`
    */
-  grid?: boolean;
+  domain?: boolean;
 
   /**
-   * A boolean flag indicating if labels should be included as part of the axis.
+   * Stroke width of axis domain line
    *
-   * __Default value:__  `true`.
+   * __Default value:__  `1`
    */
-  labels?: boolean;
+  domainWidth?: number;
 
   /**
-   * Indicates if labels should be hidden if they exceed the axis range. If `false `(the default) no bounds overlap analysis is performed. If `true`, labels will be hidden if they exceed the axis range by more than 1 pixel. If this property is a number, it specifies the pixel tolerance: the maximum amount by which a label bounding box may exceed the axis range.
+   * Color of axis domain line.
    *
-   * __Default value:__ `false`.
+   * __Default value:__  `gray`.
```

Comment:
We need to include quotes for strings (e.g., `"gray"`).  Please do the same for other properties.